tasks/install_RedHat.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - name: Install AWS CloudWatch Agent RPM via dnf
   dnf:
+    disable_gpg_check: yes
```

Comment:
We should include a comment here, similar to what you put in https://github.com/cisagov/ansible-role-amazon-ssm-agent/pull/8/commits/07efeb0a5b90e8650ed5ecfca0bdda3ed091d1f5.

Reply:
See commit 3c0461a.